lms/templates/student_account/register.underscore

```diff
@@ -5,13 +5,6 @@
 
 <form id="register" class="register-form" autocomplete="off" tabindex="-1">
 
-    <% if (context.errorMessage) { %>
```

Comment:
It would be better if you wrote a comment, or even better to comment out the stuff instead of deleting it.

This would help in future merges with edx/master.